pom.xml

```diff
@@ -364,6 +364,8 @@
                     <artifactId>maven-compiler-plugin</artifactId>
                     <version>3.3</version>
                     <configuration>
+                    	<debug>true</debug>
+                    	<debuglevel>lines,vars,source</debuglevel>
```

Comment:
Was hoping to enable debug data when compiled for remote debugging out of my IDE. <debug> is apparently 'true' by default anyway. I can keep that change for myself.